src/binaryen-c.cpp

```diff
@@ -1030,4 +1072,15 @@ void BinaryenSetAPITracing(int on) {
   }
 }
 
+//
+// ===== Emscripten support =====
+//
+
+#ifdef EMSCRIPTEN
+int __cxa_thread_atexit(void(*)(char*), char*, char*)
```

Comment:
did you figure out where this was needed? it's possibly a bug in emscripten that we should figure out